docs/fsharp/index.md

```diff
@@ -34,6 +34,14 @@ The [Tutorials](tutorials/index.md) also have step-by-step guides for various sk
 
 If you know your way around F#, you'll find a lot of use in the [Language Reference](language-reference/index.md), which documents each aspect of the language thoroughly, supplemented by numerous code samples.  You'll also find a lot of use in the [F# Core Library Reference](https://msdn.microsoft.com/en-us/visualfsharpdocs/conceptual/fsharp-core-library-reference).
 
+## The F# Software Foundation
+
+Although Microsoft is the primary developer of the F# language and Visual F# Tooling, F# is also backed by an independent foundation, the F# Software Foundation (or FSSF for short).
```

Comment:
nit: We usually just introduce the acronym in parentheses instead of saying for short.
F# Software Foundation (or FSSF for short) -> F# Software Foundation (FSSF)